packages/forms-web-app/src/pages/projects/_middleware/middleware.component.test.js

```diff
@@ -13,10 +13,7 @@ jest.mock('../../../config', () => {
 	const originalConfig = jest.requireActual('../../../config');
 	return {
 		...originalConfig,
-		featureFlag: {
-			...originalConfig.featureFlag,
-			projectMigrationCaseReferences: ['mock-case-ref-in-config']
-		}
+		featureFlag: {}
```

Comment:
Why is the `featureFlag` set to an empty object?

Reply:
Looks like config is removed entirely from packages/forms-web-app/src/pages/projects/_middleware/middleware.js

Reply:
Thank you, good spot. I've now removed config from the middleware test.